### ISSUE TITLE
The browser compatiblity information could be made more readable

### ISSUE THREAD

**Issue by simevidas** [mdn/yari#5804]
As part of the initiative to improve the readability experience (https://github.com/mdn/yari/discussions/5715), I suggest improving the readability of the browser compatibility information.

1. The icons are not consistent. If the cross is inside a red circle, and the question mark is inside a gray circle, then the check-mark should be inside a green circle.
2. The asterisk should not be in a circle because the circle is already used for the icons that represent level of support.
3. The table cells should be top-aligned, so that the icons and version numbers are property aligned with each other.

P.S. A developer on Twitter [pointed out](https://twitter.com/Tyriar/status/1507054773328961544) that using circles for all three support statuses can be an issue for people who are red/green color blind. Maybe not using circles at all and instead using different shapes would be a good idea (e..g., green check-mark ✔︎, red cross ❌, and yellow triangle ⚠️).

### Before

<img width="681" alt="Screenshot 2022-03-24 at 16 02 51" src="https://user-images.githubusercontent.com/716405/159946447-81d84c6d-9b2b-4ee5-9ad1-9195ede9bf76.png">

### After

<img width="673" alt="Screenshot 2022-03-24 at 16 23 16" src="https://user-images.githubusercontent.com/716405/159950777-08db4838-8f32-4411-8306-4201068b8efc.png">

**Comment by Andrew6rant** [comment]
I personally think that inconsistent icons can be good for accessibility, your example is super nice-looking but the circle icons can be a bit hard to tell apart at a quick glance if one has poor eyesight or red-green colorblindness.

Here is what yours looks like put through Chrome's deuteranopia color filter (the most common form of colorblindness):
![Šime_Vidas_MDN_AFTER](https://user-images.githubusercontent.com/57331134/160034046-fc0bd8b9-7744-46ac-8f3d-18ffbc92fbef.png)

I took a stab at making a mock-up, here is my attempt:

**Before:**

![Andrew_Grant_MDN_BEFORE](https://user-images.githubusercontent.com/57331134/160033278-fde5c31d-32a6-4459-a3b7-927ebb1b653e.png)

**After:**

![Andrew_Grant_MDN_AFTER](https://user-images.githubusercontent.com/57331134/160033294-d3da90aa-a698-4a61-b4e5-33c96d65c03a.png)

And, with deuteranopia filter:

![Andrew_Grant_MDN_AFTER_DEUTERANOPIA](https://user-images.githubusercontent.com/57331134/160034979-9c625acf-1e64-4529-a545-968282934cb7.png)

It's not as pretty as yours but the checkmark is visually distinguished from the X and question mark.

**Comment by OnkarRuikar** [comment]
Colored or circled, if icons in the legend look same as those inside BCD table, it'll look more consistent.

**Comment by schalkneethling** [comment]
@OnkarRuikar random question 😄 Are you on the MDN Web Docs matrix channel by any chance? https://matrix.to/#/#mdn:mozilla.org

**Comment by OnkarRuikar** [comment]
@schalkneethling joined! 😀

**Comment by caugner** [comment]
In the mean-time, we have addressed the readability of the BCD tables by using circles only for no support, unknown support and pre-release feature:

<img width="783" alt="image" src="https://user-images.githubusercontent.com/495429/201908126-57cfa890-a1d7-4bc4-93ec-20ba8469f855.png">
